package.json

```diff
@@ -11,19 +11,20 @@
     "umd"
   ],
   "scripts": {
-    "clean": "nwb clean-module && nwb clean-demo",
-    "start": "nwb serve-react-demo",
-    "test": "nwb test-react",
-    "build": "nwb build",
-    "build:watch": "nodemon -w src -x 'rm .babelrc &> /dev/null; nwb build-react-component --no-demo'",
-    "test:coverage": "nwb test-react --coverage",
-    "test:watch": "nwb test-react --server",
-    "storybook": "start-storybook -p 9001 -c .storybook",
-    "storybook:build": "build-storybook -c .storybook -o .out",
-    "demo:build": "nwb build-react-component src/index.js",
+    "build": "yarn clean && yarn compile && rollup -c",
+    "build:watch": "yarn rollup -w -c",
+    "clean": "rm -rf build/ es/ lib/",
+    "compile": "tsc -p .",
```

Comment:
I'm not sure that we need this step with rollup

Reply:
We do to get the files into `build/` -- but I'll see if we even need that anymore.

Reply:
I'm sure there's a way to configure storybook to use `babel-loader` (or something?) to load the `.ts` files directly, but I haven't figured that out yet. I'm going to stick with the `.ts + .js => build/**/*.js` configuration until there's a better solution.